core/badge-urls/make-badge-url.js

```diff
@@ -1,8 +1,6 @@
-'use strict'
-
-const { URL } = require('url')
-const queryString = require('query-string')
-const { compile } = require('path-to-regexp')
+import url from 'url'
```

Comment:
I changed this in 078db820dca8b2b6e58667c35ee853a9279d73ad, we were previously getting the error I mentioned earlier on in this PR, `[frontend] Attempted import error: 'URL' is not exported from 'url' (imported as 'URL'). warn ../core/badge-urls/make-badge-url.js`.

I _think_ the reason why it doesn't work in this specific case is because `make-badge-url` is the only backend file which happens to be imported by the frontend code. I'm guessing that the Typescript loader, which already supported `import` statements, has slightly different behaviours compared to the pure ESM loader.